fvm/src/syscalls/bind.rs

```diff
@@ -199,3 +199,4 @@ impl_bind_syscalls!(A B C D);
 impl_bind_syscalls!(A B C D E);
 impl_bind_syscalls!(A B C D E F);
 impl_bind_syscalls!(A B C D E F G);
+impl_bind_syscalls!(A B C D E F G H);
```

Comment:
Interesting. I don't think I had added this binding pattern here and nothing complained. Why would that be?